src/snowflake/cli/plugins/streamlit/commands.py

```diff
@@ -89,9 +89,21 @@ def streamlit_deploy(
     You can modify the behaviour using flags. For details check help information.
     """
     streamlit = cli_context.project_definition
+
     if not streamlit:
         return MessageResult("No streamlit were specified in project definition.")
 
+    database = streamlit.get("database")
+    schema = streamlit.get("schema")
+    options.pop("database")
+    options.pop("schema")
```

Comment:
```suggestion
    database = options.pop("database", None)
    schema = options.pop("schema", None)
```

Will do the same, but why we need pop?

Reply:
"options" do not seem to catch "database" and "schema" from `snowflake.yml`.
We need "pop" as later we gat duplicated keyword arguments:
```
    url = StreamlitManager().deploy(
        streamlit_name=streamlit["name"],
        database=database,
        schema=schema,
        ...
        **options,
    )
```

Reply:
Oh, I a see. Do we still need to pop from options?

Reply:
An alternative would be
```py
options['database'] = streamlit.get('database') or options['database']
```
options are later passed to `StreamlitManager().deploy()` as kwargs, and already have 'database' and 'schema' keys inside. I decided to pass them explicitly to increase readability, WDYT?

Reply:
Discussed on slack - parameters will be moved as separate PR